.github/tailor.yaml

```diff
@@ -1,6 +1,6 @@
 rules:
   - name: commit title
-    description: all commit titles must have a scope, be lowercase, and be no more than 72 characters
+    description: all commit titles must have a scope and be no more than 72 characters
```

Comment:
Should there be any change to the expression for this commit?

Reply:
It already allowers uppercase (It's using `[[:alnum:]]` which in Rust is defined as `([0-9A-Za-z])`)